tests-behat/grid.feature

```diff
@@ -36,3 +36,12 @@ Feature: Grid
     When I click using selector "xpath(//div[@id='grid']//tr[2]//a)"
     Then No toast should be displayed
     Then PATCH MINK the url should match "~_unit-test/grid-rowclick.php#test~"
+
+  Scenario: master checkbox
+    Given I am on "_unit-test/grid-rowclick.php"
+    When I click using selector "xpath(//div[@id='grid']//tr[2]//div.ui.child.checkbox)"
+    Then No toast should be displayed
```

Comment:
I should add a toast that display the checkbox selection to test if the selection is OK

Reply:
Done.